app/src/main/java/com/spoony/spoony/presentation/register/componet/AddMenuButtton.kt

```diff
@@ -0,0 +1,55 @@
+package com.spoony.spoony.presentation.register.componet
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.R
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import com.spoony.spoony.core.util.extension.noRippleClickable
+
+@Composable
+fun AddMenuButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Surface(
+        modifier = modifier.clip(RoundedCornerShape(8.dp)),
+        border = BorderStroke(1.dp, SpoonyAndroidTheme.colors.gray100)
+    ) {
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = modifier
```

Comment:
```suggestion
            modifier = Modifier
```
P1: 인자로 넘겨받은 modifier가 surface와 box에 이중으로 적용되고 있어요!! 둘 중 하나는 Modifier로 바꿔줘야 할 것 같네요